Microsoft.DotNet.Try.Markdown/Microsoft.DotNet.Try.Markdown.nuspec

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+    <metadata>
+        $CommonMetadataElements$
+        <language>en-US</language>
+        <dependencies>
+            <group targetFramework=".NETStandard2.0">
+                <!-- Dependencies of Microsoft.DotNet.Try.Markdown -->
+                <dependency id="Microsoft.AspNetCore.Html.Abstractions" version="$MicrosoftAspNetCoreHtmlAbstractionsVersion$" />
+                <dependency id="system.commandline.experimental" version="$SystemCommandlineExperimentalVersion$" />
+                <dependency id="Markdig" version="$MarkdigVersion$" />
+
+                <!-- Dependencies of MLS.Agent.Tools -->
+                <dependency id="Clockwise" version="$ClockwiseVersion$" />
```

Comment:
This should not have a Clockwise dependency.

Reply:
Done